Gemfile.lock

```diff
@@ -352,7 +352,7 @@ DEPENDENCIES
   byebug
   database_cleaner-active_record
   dotenv-rails
-  elasticsearch (~> 8.x)
+  elasticsearch (< 7.14)
```

Comment:
just fyi production uses 8.x but I recall having trouble installing it locally...

Reply:
oh no. i'll change it back then